app/src/main/java/org/simple/clinic/editpatient/PatientEditScreenController.kt

```diff
@@ -109,53 +111,23 @@ class PatientEditScreenController @Inject constructor(
 
   private fun mergeWithOngoingEntryPatientEntryChanges(): ObservableTransformer<UiEvent, UiEvent> {
     return ObservableTransformer { events ->
-      val patientUuids = events
-          .ofType<PatientEditScreenCreated>()
-          .map { it.patient.uuid }
-
-      val nameChanges = events
-          .ofType<PatientEditPatientNameTextChanged>()
-          .map { it.name }
-
-      val genderChanges = events
-          .ofType<PatientEditGenderChanged>()
-          .map { it.gender }
-
-      val colonyOrVillageChanges = events
-          .ofType<PatientEditColonyOrVillageChanged>()
-          .map { it.colonyOrVillage }
-
-      val districtChanges = events
-          .ofType<PatientEditDistrictTextChanged>()
-          .map { it.district }
-
-      val stateChanges = events
-          .ofType<PatientEditStateTextChanged>()
-          .map { it.state }
-
-      val phoneNumberChanges = events
-          .ofType<PatientEditPhoneNumberTextChanged>()
-          .map { it.phoneNumber }
-
       val ageChanges = events
```

Comment:
We can change `ageChanges` and `dateOfBirthChanges` to also use `mapType`, right?